packages/core/src/index.js

```diff
@@ -12,6 +12,7 @@ import {
   ACTION_FETCH_FAILURE,
   ACTION_UPDATE_LOCAL,
   ACTION_RESET_LOCAL
```

Comment:
it's actually not ok because of the missing `,`. But I fixed that

Reply:
🙇